x-pack/legacy/plugins/siem/public/pages/case/components/user_action_tree/user_action_item.tsx

```diff
@@ -23,6 +24,42 @@ interface UserActionItemProps {
   userName: string;
 }
 
+const UserAction = styled(EuiFlexGroup)`
```

Comment:
can we call it UserActionItemContainer or something else? to differentiate it with its parents